deps/wazuh_testing/wazuh_testing/remote.py

```diff
@@ -35,14 +34,14 @@
 data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
 
 
-def new_agent_group(group_name=DEFAULT_TESTING_GROUP_NAME, configuration_file='agent.conf'):
+def new_agent_group(group_name=DEFAULT_TESTING_GROUP_NAME, configuration_file=REMOTE_AGENT_CONF):
     """Create a new agent group for testing purpose, must be run only on Managers."""
 
     sb.run([f"{tools.WAZUH_PATH}/bin/agent_groups", "-q", "-a", "-g", group_name])
 
     agent_conf_path = os.path.join(data_path, configuration_file)
 
-    with open(f"{tools.WAZUH_PATH}/etc/shared/{group_name}/agent.conf", "w") as agent_conf_file:
+    with open(f"{tools.WAZUH_PATH}/etc/shared/{group_name}/{configuration_file}", "w") as agent_conf_file:
```

Comment:
Be careful with this. If you `REMOTE_AGENT_CONF` is set to `os.path.join(WAZUH_PATH, 'etc', 'shared', 'default', 'shared.conf')`, so this line will raise an `IOError` because this file: `"{tools.WAZUH_PATH}/etc/shared/{group_name}/os.path.join(WAZUH_PATH, 'etc', 'shared', 'default', 'shared.conf')}` doesn't exist.